quran/models.py

```diff
@@ -0,0 +1,24 @@
+'''Models describing abstractions used at Quran app'''
+from django.db import migrations, models
+from django.contrib.postgres.fields import ArrayField
+
+
+class Surah(models.Model):
+    '''General information about Surahs'''
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=20)
+    total_ayahs = models.PositiveIntegerField(default=0)
+
+
+class Ayah(models.Model):
+    '''General information about Ayah'''
+    id = models.AutoField(primary_key=True, serialize=False)
```

Comment:
Don't convert an `id` to a `primary_key`. Just remove that `primary_key=True` and `serialize=False`.

Reply:
You can actually remove the whole line. `id` field will be added automatically: https://docs.djangoproject.com/en/2.1/topics/db/models/#automatic-primary-key-fields